regression/pages/studio/course_outline_page.py

```diff
@@ -98,3 +99,12 @@ def delete_section(self):
         ).first.click()
         self.wait_for_ajax()
         click_confirmation_prompt_primary_button(self)
+
+    def click_edit_start_date_button(self):
+        """
+        Clicks edit start date button on Course Outline page
+        """
+        button = self.q(css='.action-button span.icon.fa.fa-pencil').results[0]
+        ActionChains(
```

Comment:
Could you add a comment describing why you need an `ActionChain` to perform the click?